tests/view_tests/test_image_routes.py

```diff
@@ -34,8 +29,10 @@ def test_post_image_route_with_file_bad_extension(client):
     assert response.status_code == 400
     assert b'Error with request: File extension not allowed.' in response.data
 
-def test_post_image_route_with_file(client):
+@patch('werkzeug.datastructures.FileStorage.save')
```

Comment:
You should use the [monkeytest fixture](https://docs.pytest.org/en/latest/monkeypatch.html) built into pytest instead. I must have accidentally added mock but we don't actually need it as a dependency. I already used monkeytest in my logger tests so better to be consistent. The calls are very similar though so should be easy to switch to. You can use a lambda with monkeypatch for the return value.

Reply:
See docs for a proper [lambda example](https://docs.pytest.org/en/latest/reference.html#_pytest.monkeypatch.MonkeyPatch.setattr).